### ISSUE TITLE
Fails with latest version of leaflet and react-leaflet

### ISSUE THREAD

**Issue by cherouvim** [LiveBy/react-leaflet-control#24]
I updated these:
```
-    "leaflet": "1.3.1",
+    "leaflet": "1.3.3",
-    "react-leaflet": "1.9.1",
+    "react-leaflet": "2.0.0",
```
and it now fails with:
```
Uncaught Error: createLeafletElement() must be implemented
```

**Comment by kevinhughes27** [comment]
Looks like this is fixed in #25 if it gets merged...

**Comment by connordittmar** [comment]
For those who are still waiting on #25, a quick fix to work with react-leaflet 2.0:

1. Add a (near) dummy function to return the element created in `componentWillMount`

2. Add the context exporter (new to 2.0 I believe) to the default export:

```javascript
# in imports:
import { MapControl, withLeaflet } from 'react-leaflet'

...

class Control extends MapControl {
... 

  createLeafletElement () {
    return this.leafletElement
  }

...

}

export default withLeaflet(Control)
```
